cpp/include/Ice/SSL/Config.h

```diff
@@ -23,6 +23,18 @@
 #        undef SECURITY_KERNEL
 #    endif
 #    define SECURITY_WIN32 1
+
+// See https://learn.microsoft.com/en-us/windows/win32/api/schannel/ns-schannel-sch_credentials#remarks
+#    define SCHANNEL_USE_BLACKLISTS
```

Comment:
Maybe we should include a header instead?

This header is a public header, so if you define a macro like SCHANNEL_USE_BLACKLISTS, you need to protect this definition against redefinition.

Reply:
Fixed, and use a header instead.